tests/test_mgxs_library_nuclides/test_mgxs_library_nuclides.py

```diff
@@ -25,8 +25,12 @@ def _build_inputs(self):
         # Initialize MGXS Library for a few cross section types
         self.mgxs_lib = openmc.mgxs.Library(self._input_set.geometry)
         self.mgxs_lib.by_nuclide = True
-        self.mgxs_lib.mgxs_types = ['transport', 'nu-fission',
-                                    'nu-scatter matrix', 'chi']
+        self.mgxs_lib.mgxs_types = ['total', 'transport', 'nu-transport',
```

Comment:
How about using `MGXS_TYPES` so that these tests will "scale" with the introduction of `MGXS` subclasses in the future:

```python
self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES
```

Though I can see the disadvantage of this being that the "mgxs_*_nuclide" tests will *always* fail each time a new `MGXS` subclass is introduced.